### PR TITLE
XAMARIN-43636 - Linking Properties, Methods and Events to SfSwitch

### PR DESCRIPTION
**Pull request by shivaniramakrishnan14**
Linking Properties, Methods and Events to SfSwitch

### CODE REVIEW

**Comment by shivaniramakrishnan14** [comment]
rebuild

**Comment by shivaniramakrishnan14** [comment]
rebuild

**Comment by shivaniramakrishnan14** [comment]
rebuild

File: Xamarin/Switch/how-to.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+description: Explaining about different states in Syncfusion Switch Control (SfSwitch) for Xamarin.Forms and how it performs.
 platform: Xamarin
 control: Switch
 documentation: ug
 ---
 
-# How to
+# How to Add States in Xamarin Switch (SfSwitch)
 
 ## Show busy indicator to perform async action
```
Comment (Choza-rajan) [inline Xamarin/Switch/how-to.md:12]: How to show busy indicator to perform async action

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**